src/controller/subtitle-stream-controller.ts

```diff
@@ -326,6 +326,10 @@ export class SubtitleStreamController
               tdecrypt: endTime,
             },
           });
+        })
+        .catch((e) => {
+          this.warn(e.message);
```

Comment:
Recently, in #4933 the exception I encountered had no message, only an `e.name` of `"OperationError"`. Should we account for that here?

```suggestion
          this.warn(e.name || e.message);
```

Reply:
You are talking about a WebCrypto error which is handled by 
```
 .catch((err) => {
        logger.warn(
          '[decrypter.ts]: WebCrypto Error, disable WebCrypto API:',
          err
        );
```